src/Search/Type/Base.php

```diff
@@ -1,9 +1,9 @@
 <?php
-namespace Search\Type;
+namespace Search\Seach\Type;
 
 use Cake\ORM\Query;
 use Cake\Core\InstanceConfigTrait;
-use Search\Manager;
+use Search\Search\Manager;
```

Comment:
OK, but you're aware this will become `namespace FOC\Search\Search` for the search related classes then?